javascript/javascript2/week2/Jsweek2.js

```diff
@@ -0,0 +1,183 @@
+ // Flight booking fullname function
+ 
+ 
+  let  fullname1 = ["Benjamin", "Hughes", true,"male"];
+  let fullname2 = ["Lukas", "Andrias",false,"male"]; 
+  let fullname3 = ["Sofia", "Larsen",true,"female"]; 
+   function getFullname (firstname, surname) {  
+    if (firstname=="" || surname=="" || typeof firstname !== "string" ||
+         typeof surname !== "string"){
+    return `Please enter a valid name.`
+       }
+    return  ` ${firstname} ${surname}` ; 
+ }
+ console.log(getFullname(fullname1[0], fullname1[1]));
+ console.log(getFullname(fullname2[0], fullname2[1])); 
```

Comment:
You should have two variables that you assigned to calling the getFullname function, here you should log variables. The assignment says: `And the assignment says: `Now try to create two variables fullname1 and fullname2 these two variables should be assigned to calling the getFullname function.`
`
```suggestion
const fullname1 = getFullname(personData1[0], personData1[1]);
const fullname1 = getFullname(personData2[0], personData2[1]);
console.log(fullname1);
console.log(fullname2); 
```